p/themes/Alternative-Dark/template.css

```diff
@@ -869,18 +869,13 @@ br {
 }
 
 /*=== "Load" parts */
-#stream {
-	position: relative;
-	z-index: 5;
-}
 
 /* Only matches while waiting for div#stream to be received */
 #first_load:nth-last-of-type(2) {
 	margin: -40px auto 0 auto;
 	height: 40px;
 	position: relative;
-	top: 250px;
-	z-index: 2;
+	top: 290px;
```

Comment:
I don't see why the position relative and `top` are needed either :thinking:

Reply:
I think the problem comes from the margin `-40px`, you just have to use a positive value :)

Reply:
For that to work, we would need to move `#first_load` to after `.nav_menu` in the DOM, which increases the risk of not being able to benefit the load effect, because `.nav_menu` is quite large and therefore significant in the buffering

Positive margin, worse:
![image](https://user-images.githubusercontent.com/1008324/77945257-a3771880-72c0-11ea-8132-1cdfa03701e5.png)

Example without `.nav_menu` loaded yet:
![image](https://user-images.githubusercontent.com/1008324/77946323-67dd4e00-72c2-11ea-9f82-1413ea828f32.png)

Reply:
I hadn't thought of that!

I think a cleaner option would be to use the CSS grid system, but it's less supported and would require more work, so let's go with yours for now :)

Reply:
And for some reason, Chrome does not behave as I believe it should (not rendering the animation like Firefox or Edge). I am looking at it as part of https://github.com/FreshRSS/FreshRSS/pull/2909

Reply:
Done https://github.com/FreshRSS/FreshRSS/pull/2909/commits/1e627dd5554cf80c3bac480ef261a4e6d5bfa093